pkg/logs/sds/rules.go

```diff
@@ -26,12 +26,17 @@ type MatchAction struct {
 
 // StandardRuleConfig as sent by the Remote Configuration;
 type StandardRuleConfig struct {
-	ID                      string               `json:"id"`
-	Name                    string               `json:"name"`
-	Description             string               `json:"description"`
+	ID          string                   `json:"id"`
+	Name        string                   `json:"name"`
+	Description string                   `json:"description"`
+	Definitions []StandardRuleDefinition `json:"definitions"`
+}
+
+// StandardRuleDefinition contains a versioned standard rule definition.
+type StandardRuleDefinition struct {
+	Version                 int                  `json:"version"`
 	Pattern                 string               `json:"pattern"`
 	Tags                    []string             `json:"tags"`
-	Type                    string               `json:"type"`
```

Comment:
I see we dropped `Type` here too. Was it incidentally unused? Or is it related to this change?

Reply:
Incidentally unused.